features/wijzigingen.feature

```diff
@@ -0,0 +1,100 @@
+
+    Scenario: Gebruiker heropent de volgindicatie en wijzigingen vóór het heropenen worden niet gemeld
+        Gegeven gebruiker "A" op 17 januari 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992351" met einddatum "2021-03-01"
+        En op 21 februari 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999992351"
+        En gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992351" en geen einddatum
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999992351"
+
+    Scenario: Gebruiker vraagt expliciet wijzigingen van vóór het zetten van de volgindicatie
+        Gegeven op 4 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999992971"
+        En gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992971" en geen einddatum
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen?vanaf=2021-03-01
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999992971"
+
+    Scenario: Gebruiker verlengt de volgindicatie en eerdere wijzigingen worden wel gemeld
+        Gegeven gebruiker "A" op 17 januari 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "000009945" met einddatum "2021-03-05"
+        En op 21 februari 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "000009945"
+        En gebruiker "A" op 4 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "000009945" en geen einddatum
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" een item met de waarde "000009945"
+
+    Scenario: Meerdere gebruikers volgen zelfde persoon met verschillende einddatum
+        Gegeven gebruiker "A" op 17 januari 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999990962" met einddatum "2021-03-10"
+        En gebruiker "B" op 2 februari 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999990962" en geen einddatum
+        En op 12 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999990962"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        En gebruiker "B" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord voor gebruiker "A" in "burgerservicenummers" GEEN item met de waarde "999990962"
+        En bevat het antwoord voor gebruiker "B" in "burgerservicenummers" een item met de waarde "999990962"
+
+    Scenario: Gebruiker krijgt wijzigingen van de dag dat wijzigingen worden gevraagd
```

Comment:
Wat wordt bedoeld met `Gebruiker krijgt wijzigingen van de dag dat wijzigingen wordt gevraagd`? De gebruiker vraagt op 4 maart en geeft geen vanaf datum op. Is de vanaf datum dan 4 maart of 17 januari. Beide gevallen geven in dit geval dezelfde response terug. Ik denk dat het handig is om in een dan stap aan te geven welke datum als vanaf datum wordt gebruikt.